lib/typescript/render/components/RichCard/index.tsx

```diff
@@ -51,27 +59,58 @@ export const RichCard = ({title, description, suggestions, media, cardWidth, com
   };
 
   return (
-    <div className={styles.richCardContainer} style={cardWidth === 'SHORT' ? {width: '136px'} : {width: '280px'}}>
-      <div className={styles.mediaContainer}>
-        <Media {...media} />
-      </div>
-      <div className={styles.textContainer}>
-        {title && <h1 className={styles.title}>{title}</h1>}
-        {description && <span className={styles.description}>{description}</span>}
-        <div className={styles.suggestionsContainer}>
-          {suggestions.map((suggestion, idx) => (
-            <button
-              type="button"
-              key={idx}
-              className={styles.suggestionButton}
-              onClick={() => {
-                clickSuggestion(suggestion);
-              }}>
-              {suggestion.reply ? suggestion.reply.text : suggestion.action.text}
-            </button>
-          ))}
+    <>
+      {fromContact ? (
```

Comment:
Maybe we could build it in a way that it's not longer needed to differentiate that stuff in the components?

Reply:
Do you have any good solution for this already in mind? @bitboxer

Reply:
the `div` surrounding the component could be a flex layout and use the flex attributes to render something on the left or right?

Reply:
I agree with @bitboxer: From the looks of it the only differentiator between the ternary branches is a `flex` attribute. So since everything is supposed to be wrapped in `MessageInfo` (good name btw 👍 ) it should go there so we can completely remove the `fromContact` dependency.

Reply:
then we need `MessageInfo` also in the chat plugin. Because the chat plugin also need to render messages in both sides.